gateway/webhook/webhook_test.go

```diff
@@ -518,3 +518,96 @@ func TestPrepareRequestBody(t *testing.T) {
 		})
 	}
 }
+
+func TestAllowGetReqForWebhookSrc(t *testing.T) {
```

Comment:
Can we add it as use case on our webhook integration tests ?

Reply:
Yes. I will add it